cmd/kured/pflags.go

```diff
@@ -1,31 +1,34 @@
 package main
 
 import (
+	"fmt"
 	"regexp"
 )
 
+// Custom flag type to handle *regexp.Regexp with pflag
 type regexpValue struct {
 	value **regexp.Regexp
```

Comment:
is this a missed opportunity to reconsider our "pointer to a pointer" approach here? :)

Reply:
Totally. Probably warrant having a looksie later tomorrow. Tonight is time for bed.

FYI I tried my best to keep this MR as "refactor only", and do slightly bigger changes with possible behaviour changes in the other PR.

Reply:
I pushed a new version including this fix. I would love it to stay as a separate commit.